### ISSUE TITLE
Missing labels for inputs

### ISSUE THREAD

**Issue by Hannah-goodridge** [sanusart/react-dropdown-select#190]
When using an accessibility tool for this search input (wave), it raises an issue for a missing labels for the inputs:

```
<input tabindex="-1" class="react-dropdown-select-input css-14e9a48-InputComponent e11wid6y0" size="33"  value="">
// and again for
<input tabindex="-1" name="value" value="" style="opacity: 0; width: 0px; position: absolute;">
```

**Comment by sanusart** [comment]
This is just the input. Label is outside

**Comment by Hannah-goodridge** [comment]
I see, is there a way to associate this label to the inputs?

I've tried passing an ID to the search and also seeing if passing a name prop would work but  on both occasions there is no for attribute on the label, so it still fails accessibility.

**Comment by sanusart** [comment]
Wrapping a <label/> around should solve the case according to docs.
https://www.w3.org/WAI/tutorials/forms/labels/#associating-labels-implicitly
